mapclientplugins/geometricfitstep/view/geometricfitwidget.py

```diff
@@ -514,48 +501,49 @@ def _updateGroupSettingWidgets(self):
         isConfig = isinstance(self._currentFitterStep, FitterStepConfig)
         isFit = isinstance(self._currentFitterStep, FitterStepFit)
         if isConfig:
-            self._updateCentralProjection()
-            self._updateDataProportion()
+            self._updateGroupConfigCentralProjection()
+            self._updateGroupConfigDataProportion()
         elif isFit:
-            self._updateDataWeight()
-            self._updateStrainPenalty()
-            self._updateCurvaturePenalty()
-        self._ui.configCentralProjection_checkBox.setVisible(isConfig)
-        self._ui.configSetCentralProjection_checkBox.setVisible(isConfig)
-        self._ui.configDataProportion_checkBox.setVisible(isConfig)
-        self._ui.configDataProportion_lineEdit.setVisible(isConfig)
-        self._ui.configDataWeight_checkBox.setVisible(isFit)
-        self._ui.configDataWeight_lineEdit.setVisible(isFit)
-        self._ui.configStrainPenalty_checkBox.setVisible(isFit)
-        self._ui.configStrainPenalty_lineEdit.setVisible(isFit)
-        self._ui.configCurvaturePenalty_checkBox.setVisible(isFit)
-        self._ui.configCurvaturePenalty_lineEdit.setVisible(isFit)
-
-    def _configSettingGroupChanged(self, index):
+            self._updateGroupFitDataWeight()
+            self._updateGroupFitStrainPenalty()
+            self._updateGroupFitCurvaturePenalty()
+        self._ui.groupConfigCentralProjection_checkBox.setVisible(isConfig)
+        self._ui.groupConfigSetCentralProjection_checkBox.setVisible(isConfig)
+        self._ui.groupConfigDataProportion_checkBox.setVisible(isConfig)
+        self._ui.groupConfigDataProportion_lineEdit.setVisible(isConfig)
+        self._ui.groupFitDataWeight_checkBox.setVisible(isFit)
+        self._ui.groupFitDataWeight_lineEdit.setVisible(isFit)
+        self._ui.groupFitStrainPenalty_checkBox.setVisible(isFit)
+        self._ui.groupFitStrainPenalty_lineEdit.setVisible(isFit)
+        self._ui.groupFitCurvaturePenalty_checkBox.setVisible(isFit)
+        self._ui.groupFitCurvaturePenalty_lineEdit.setVisible(isFit)
+
+    def _groupSettingChanged(self, index):
         """
         Callback for change in setting group field chooser widget.
         """
         self._updateGroupSettingWidgets()
 
-    def _getGroup(self):
-        group = self._ui.configSettingGroup_fieldChooser.getField()
+    def _getGroupSettingsGroupName(self):
+        group = self._ui.groupSettings_fieldChooser.getField()
+        groupName = None
         if group:
-            group = group.getName()
-        return group
+            groupName = group.getName()
+        return groupName
 
     def _getGroupSettingDisplayState(self, func):
-        group = self._getGroup()
+        group = self._getGroupSettingsGroupName()
```

Comment:
Update all callers to this method to assign variable as:
groupName = self._getGroupSettingsGroupName()